src/lib/server/firebaseUtils.ts

```diff
@@ -147,9 +147,20 @@ export const listAllUsers = async (nextPageToken?: string): Promise<UserRecord[]
   }
   return users;
 };
-
 export const getUserFromEmail = (email: string) => auth.getUserByEmail(email);
 
+export const deleteUser = async (uid: string) => {
```

Comment:
Not sure if this is handled elsewhere but I think we also want to delete the `/flags/user/${uid}` path?